diff -cr usb_cam/launch/camera.launch.py ../backup/usb_cam/launch/camera.launch.py
*** usb_cam/launch/camera.launch.py	2025-08-25 18:36:53.023539589 -0400
--- ../backup/usb_cam/launch/camera.launch.py	2025-08-25 17:20:34.815147864 -0400
***************
*** 68,74 ****
              package='usb_cam', executable='usb_cam_node_exe', output='screen',
              name=camera.name,
              namespace=camera.namespace,
!             parameters=[camera.param_path],
              remappings=camera.remappings
          )
          for camera in CAMERAS
--- 68,76 ----
              package='usb_cam', executable='usb_cam_node_exe', output='screen',
              name=camera.name,
              namespace=camera.namespace,
!             parameters=[{'video_device': '/dev/video0'},
!                         {'pixel_format': 'mjpeg2rgb'},
!                         {'camera_info_url': '/workspaces/isaac_ros-dev/src/camera_info.yaml'}],
              remappings=camera.remappings
          )
          for camera in CAMERAS
Only in usb_cam/launch: camera.launch.py.orig
Only in usb_cam/launch: camera.launch.py.rej
